fares-data-build-tool/fdbt-site
# feat: school services on confirmation pages

# Description

-   This PR adds consideration of school tickets to the fareConfirmation, ticketConfirmation and salesConfirmation pages/APIs.

# Testing instructions

-   Pull down this branch and run the site locally. Validate that (i) all original user journeys, and (ii) all school service ticket user journeys work as expected and the confirmation pages display the correct information.

# Type of change


# Checklist:
